WebKitBrowser/Extension/CMakeLists.txt

```diff
@@ -60,9 +61,19 @@ if(PLUGIN_WEBKITBROWSER_BADGER_BRIDGE)
     target_compile_definitions(${MODULE_NAME} PRIVATE ENABLE_BADGER_BRIDGE)
 endif()
 
-if(PLUGIN_WEBKITBROWSER_FIREBOLTOS_ENDPOINT)
-    target_sources(${MODULE_NAME} PRIVATE FireboltOSEndpoint.cpp)
-    target_compile_definitions(${MODULE_NAME} PRIVATE ENABLE_FIREBOLTOS_ENDPOINT)
+if(PLUGIN_WEBKITBROWSER_UPDATE_TZ_FROM_FILE)
```

Comment:
Isn't this already part f the TimeSync plugin? Is this just because Comcast works with Monolithic blobs and do not adhere to the modular design? If so could we add a comment that this is a Comcast only feature and for a modular architecture (sepertion of concern) they should just use the TimeSync ?

Reply:
comments added and changed the default value of PLUGIN_WEBKITBROWSER_UPDATE_TZ_FROM_FILE flag